### PR TITLE
feat: Add transmitter info into BatchJournalRequestEntry

### PR DESCRIPTION
**Pull request by hiohiohio**
Context:
- transmitter info is required in some cases to process journal requests

Changes:
- transmitter info is added into BatchJournalRequestEntry
- add description into BatchJournalRequestEntry

### CODE REVIEW

**Comment by RayOconnor** [comment]
Thanks very much hiohiohio! Very excited to use this feature as soon as it's merged.

File: alpaca/broker/requests.py [hunk 1]
```diff
@@ -891,11 +891,21 @@ class BatchJournalRequestEntry(NonEmptyRequest):
...
     """
 
     to_account: UUID
-    amount: float
+    amount: Union[str, float]
+    transmitter_name: Optional[str] = None
+    transmitter_account_number: Optional[str] = None
+    transmitter_address: Optional[str] = None
+    transmitter_financial_institution: Optional[str] = None
+    transmitter_timestamp: Optional[str] = None
```
Comment (RayOconnor) [inline alpaca/broker/requests.py:908]: Does this endpoint also accept a `description` per individual journal in the batch, similar to the [single create journal endpoint](https://docs.alpaca.markets/reference/createjournal-1)?

Comment (hiohiohio) [reply alpaca/broker/requests.py:908]: Good catch! Please let me check and add it if it is acceptable.

**Review by said-saifi — APPROVED** [review]
LGTM

File: alpaca/broker/requests.py (hunk 1 above)
Comment (hiohiohio) [reply alpaca/broker/requests.py]: added!

**Pull request merged**